### DISCUSSION TITLE
GitHub Projects is AWESOME!

### DISCUSSION THREAD

**Discussion by MakePixelsWork** [community/community#125420] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

Instead of posting bugs, feature requests or other updates everyone wants... I thought it was time to post a big **thank you** to GitHub for providing us with such a wonderful project management tool. The free version is chuck full of features and trumps a lot of the other systems I've used over the years. 

So... here goes: 

### THANK YOU! YOU ARE AWESOME!

_Upvote if you agree._

One of the projects I help manage over at my job. Just look how beautiful this can be ordered. Lots more freedom than other tools... and no unneeded clutter. I'm in love!

![image](https://github.com/community/community/assets/56885246/1a259615-44e2-4e22-a069-5101dc978f77)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by azenMatt** [comment, marked as answer]
Thanks @MakePixelsWork 😄 made my day!
